tdb/vidrl_upload.py

```diff
@@ -49,6 +49,7 @@ def convert_vidrl_xls_to_tsv(path, fstem, ind, assay_type):
     serum_id_pattern = r"^[A-Z]\d{4,8}"
     serum_passage_pattern = r"(MDCK\d+|SIAT\d+|E\d+)"
     serum_abbrev_pattern = r"\w+\s{0,1}\w+/\d+.*"
+    serum_clade_pattern = r"^\w+\.\w+[\w\.]*"
```

Comment:
Sigh, running into a workbook where this clade pattern matches multiple rows and it ends up using the row _without_ the egg/cell labels for the human sera, so the column gets skipped. 

Will think on this some more...

Reply:
Rather than using the voting system to parse the clade row, I'm just including an extra row of data for the human serum data:

https://github.com/nextstrain/fauna/blob/e32d1f90d7ffb7c99dff1a471df43cc4cc9dc222/tdb/titer_block.py#L293-L294